gwswlib/tests/test_exporter.py

```diff
@@ -26,6 +28,78 @@ def test_check_connection_db(caplog):
     assert session is not None
 
 
+def test_import_hydx_unknown_connection_types(caplog):
+    # TODO ik zou graag in één keer die hydx.connections willen maken zonder .import_csvline te gebruiken. Is dat mogelijk?
+    # daarbij zou ik de line_out uit test_check_init_connection in test.hydx.py willen gebruiken.
+    unknown_connection_line = OrderedDict(
+        [
+            ("UNI_IDE", "ovs1"),
+            ("KN1_IDE", "knp1"),
+            ("KN2_IDE", "knp2"),
+            ("VRB_TYP", "XXX"),
+            ("LEI_IDE", ""),
+            ("BOB_KN1", ""),
+            ("BOB_KN2", ""),
+            ("STR_RCH", "OPN"),
+            ("VRB_LEN", ""),
+            ("INZ_TYP", ""),
+            ("INV_KN1", ""),
+            ("UTV_KN1", ""),
+            ("INV_KN2", ""),
+            ("UTV_KN2", ""),
+            ("ITO_IDE", ""),
+            ("PRO_IDE", ""),
+            ("STA_OBJ", ""),
+            ("AAN_BB1", ""),
+            ("AAN_BB2", ""),
+            ("INI_NIV", ""),
+            ("ALG_TOE", ""),
+        ]
+    )
+    csvfilename = "Verbinding1.csv"
+    hydx = Hydx()
+    hydx.import_csvline(unknown_connection_line, csvfilename)
+    threedi = Threedi()
+    threedi.import_hydx(hydx)
+    assert '"typeverbinding" is not recognized' in caplog.text
```

Comment:
Vraag is misschien of dit een waarschuwing moet zijn waarop je checkt of dat dit deel van de import niet gewoon een error moet raisen?
(Die error kan je een niveautje hoger dan wel weer afvangen en loggen. Op het niveau waar je hier test was een reguliere exceptie handiger/gebruikelijker.)